.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
-	branch = bleeding
+	url = https://github.com/sanman00/SpongeAPI.git
+	branch = sleeping-data-impl
```

Comment:
minecrell introduced a system at some point where this is not necessary anymore.